packages/kitchen/src/pan/Treeviews.vue

```diff
@@ -53,14 +52,14 @@
                 open-on-click
                 transition
               >
-                <v-icon
-                  v-if="!item.children"
-                  slot="prepend"
-                  slot-scope="{ item, active }"
-                  :color="active ? 'primary' : ''"
-                >
-                  mdi-account
-                </v-icon>
+                <template #prepend="{ item, active }">
+                  <v-icon
+                    v-if="!item.children"
```

Comment:
Should the `if` be moved to `<template>`?

Reply:
it doesn't work this way